### PR TITLE
Cli: cosmetics

### PR DESCRIPTION
**Pull request by sergiogiro**

### CODE REVIEW

File: test/configuration/specify-configuration-path/expected-logs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+=> Capturing the files in /data/app
```
Comment (SamirTalwar) [inline test/configuration/specify-configuration-path/expected-logs]: Where in this test does it actually specify this?

Comment (sergiogiro) [reply test/configuration/specify-configuration-path/expected-logs]: The test specifies the configuration path, not the context. All tests have the same context path, I changed this for all tests.

Comment (SamirTalwar) [reply test/configuration/specify-configuration-path/expected-logs]: OK, fair. So why does this PR add an entire set of logs for this test, but only change one line for the others?

Comment (sergiogiro) [reply test/configuration/specify-configuration-path/expected-logs]: configuration path = path to plz.config.json

Comment (sergiogiro) [reply test/configuration/specify-configuration-path/expected-logs]: Oh, that file was uploaded inadvertently. I couldn't write that test with the current test infra. There's no test at all

Comment (sergiogiro) [reply test/configuration/specify-configuration-path/expected-logs]: Good catch :P

**Pull request merged**